service/src/main/java/skills/skillLoading/SkillsLoader.groovy

```diff
@@ -974,6 +974,17 @@ class SkillsLoader {
                         numSkillsRequired: skillDef.numSkillsRequired,
                         totalPoints: skillDef.totalPoints,
                 )
+
+                Boolean groupDescriptionsOn = settingsService.getProjectSetting(skillDef.projectId, 'group-descriptions')?.value?.toBoolean()
```

Comment:
suggesting to add this setting to the ``skills.services.settings.Settings`` enum